worlds/sc2/Items.py

```diff
@@ -68,175 +241,727 @@ def get_full_item_list():
+                 description="Doubles the width of Hellion's flame attack."),
+    ItemNames.HELLION_THERMITE_FILAMENTS:
+        ItemData(301 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 1, SC2Race.TERRAN,
+                 parent_item=ItemNames.HELLION,
+                 description="Hellions do additional 10 damage to Light Armor."),
+    ItemNames.SPIDER_MINE_CERBERUS_MINE:
+        ItemData(302 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 2, SC2Race.TERRAN,
+                 classification=ItemClassification.filler,
+                 description="Increases trigger and blast radius of Spider Mines."),
+    ItemNames.VULTURE_REPLENISHABLE_MAGAZINE:
+        ItemData(303 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 3, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.VULTURE,
+                 description="Allows Vultures to replace used Spider Mines. Costs 15 minerals."),
+    ItemNames.GOLIATH_MULTI_LOCK_WEAPONS_SYSTEM:
+        ItemData(304 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 4, SC2Race.TERRAN,
+                 parent_item=ItemNames.GOLIATH,
+                 description="Goliaths can attack both ground and air targets simultaneously."),
+    ItemNames.GOLIATH_ARES_CLASS_TARGETING_SYSTEM:
+        ItemData(305 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 5, SC2Race.TERRAN,
+                 parent_item=ItemNames.GOLIATH,
+                 description="Increases Goliath ground attack range by 1 and air by 3."),
+    ItemNames.DIAMONDBACK_TRI_LITHIUM_POWER_CELL:
+        ItemData(306 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 6, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.DIAMONDBACK,
+                 description="Increases Diamondback attack range by 1."),
+    ItemNames.DIAMONDBACK_SHAPED_HULL:
+        ItemData(307 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 7, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.DIAMONDBACK,
+                 description="Increases Diamondback life by 50."),
+    ItemNames.SIEGE_TANK_MAELSTROM_ROUNDS:
+        ItemData(308 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 8, SC2Race.TERRAN,
+                 classification=ItemClassification.progression, parent_item=ItemNames.SIEGE_TANK,
+                 description="Siege Tanks do additional 40 damage to primary target in Siege Mode."),
+    ItemNames.SIEGE_TANK_SHAPED_BLAST:
+        ItemData(309 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 9, SC2Race.TERRAN,
+                 parent_item=ItemNames.SIEGE_TANK,
+                 description="Reduces splash damage to friendly targets while in Siege Mode by 75%."),
+    ItemNames.MEDIVAC_RAPID_DEPLOYMENT_TUBE:
+        ItemData(310 + SC2WOL_ITEM_ID_OFFSET, "Armory 2", 10, SC2Race.TERRAN,
+                 classification=ItemClassification.filler, parent_item=ItemNames.MEDIVAC,
+                 description="The Medivac deploys loaded troops almost instantly."),
```

Comment:
3 of the next 4 descriptions are inconsistent with how they refer to units. To make them consistent:
> *Medivacs deploy* loaded troops almost instantly.

> _Medivacs_ can heal two targets at once.

> _Wraiths evade_ 20% of incoming attacks while cloaked.